ui/dashboards/src/components/SaveDashboardButton/SaveDashboardButton.tsx

```diff
@@ -74,7 +77,9 @@ export const SaveDashboardButton = ({ onSave, isDisabled, variant = 'contained'
       } catch (error) {
         throw new Error(`An error occurred while saving the dashboard. ${error}`);
       } finally {
-        setSavingDashboard(false);
+        if (isSavingDashboard) {
```

Comment:
This line is the fix for the `Can't perform a React state update on an unmounted component.` console warning when saving using gql